packages/@okta/vuepress-site/docs/reference/api/oauth-clients/index.md

```diff
@@ -62,8 +62,7 @@ curl -v -X POST \
      "id_token"
   ],
   "grant_types": [
-     "authorization_code",
-     "refresh_token"
```

Comment:
We should not remove this, we should instead add `implicit` to the list of grant types.